sys/src/go/cmd/tmpfs/tmpfs.go

```diff
@@ -0,0 +1,344 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sync"
+
+	"harvey-os.org/internal/tmpfs"
+	"harvey-os.org/pkg/ninep"
+	"harvey-os.org/pkg/ninep/protocol"
+)
+
+var (
+	networktype = flag.String("ntype", "tcp4", "Default network type")
```

Comment:
could it just be "net", that's kind of consistent with the Go nomenclatures
tmps -net=tcp
etc.etc.

Reply:
net clashes with the "net" import

Reply:
go pkg uses "network" as the arg name, so could use that (here and for ufs for consistency)

Reply:
or just use net as the flag name and keep the var name as is